COPYRIGHT.txt

```diff
@@ -508,7 +508,6 @@ Comment: Vulkan Headers
 Copyright: 2014-2023, The Khronos Group Inc.
   2014-2023, Valve Corporation
   2014-2023, LunarG, Inc.
-  2015-2023, Google Inc.
```

Comment:
Why was this removed? 

Google inc. is still a valid contributor to the Vulkan " Khronos Group Inc."  Vulkan SDK 1.3.268.0  falls into the version category date of: " October 24, 2023 " 

2024 version is not released, it would not be valid to remove or change it.

Reply:
Google Inc. was in copyright list because it was present on `vk_enum_string_helper.h` [when it was on Vulkan-ValidationLayers](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/628cd310bef6d54b4e6b25b5ac2ed013473409d6/layers/vulkan/generated/vk_enum_string_helper.h#L9).

After it was moved to Vulkan-Utility-Libraries, [it got removed](https://github.com/KhronosGroup/Vulkan-Utility-Libraries/blob/79d5fb108be1a013135d3aa9a88c38f0b0608e87/include/vulkan/vk_enum_string_helper.h#L6).

After this move, no mentions of Google are left in copyright statements, so I thought it made sense to remove.